mithril-common/src/entities/protocol_message.rs

```diff
@@ -21,6 +21,12 @@ pub enum ProtocolMessagePartKey {
     #[serde(rename = "next_aggregate_verification_key")]
     NextAggregateVerificationKey,
 
+    /// The ProtocolMessage part key associated to the Next epoch protocol parameters
+    /// The protocol parameters that will be allowed to be used to sign during the next epoch
```

Comment:
Don't forget that this use a markdown like formatting ;).
```suggestion
    /// The ProtocolMessage part key associated to the Next epoch protocol parameters
    ///
    /// The protocol parameters that will be allowed to be used to sign during the next epoch
```

Reply:
Fixed :ok_hand: